src/Centreon/Domain/HostConfiguration/HostGroupService.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
Be careful when copying and pasting the code.
What is the purpose of this class, where do you use it?